packages/provider-elasticsearch/src/example-types/filters/tagsQuery.js

```diff
@@ -122,5 +157,5 @@ export default {
   filter,
   validContext,
   buildResultQuery,
-  result,
-}
+  result: result(options?.generateKeywords || (() => [])),
```

Comment:
You've got local changes for this based on what we discussed, but the short version is to move the default logic to the place where its used (and actually just use maybeCall to not worry about it)